packages/panels/src/global_helpers.php

```diff
@@ -4,6 +4,10 @@
 use Filament\FilamentManager;
 
 if (! function_exists('filament')) {
+    /**
+     * @param string|null $plugin
```

Comment:
Do we need this line for the other to work? If not please remove it

Reply:
nah we don't i'll remove it